sourced/ml/models/df.py

```diff
@@ -12,12 +12,14 @@ class DocumentFrequencies(Model):
     """
     NAME = "docfreq"
 
-    def construct(self, docs: int, tokfreqs: Union[Iterable[Dict[str, int]], Dict[str, int]]):
+    def construct(self, docs: int, tokfreqs: Union[Iterable[Dict[str, int]],
+                  Dict[str, int]], quant_map: Dict[str, int]=None):
```

Comment:
No need to extend this interface, pls leave as is

The key prefix allows to tell the identity